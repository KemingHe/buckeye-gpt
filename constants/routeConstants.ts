export const HOME_ROUTE = '/';

// Stack Auth routes.
export const AUTH_SIGN_IN_ROUTE = '/auth/sign-in';
export const AUTH_VERIFY_EMAIL_ROUTE = '/auth/verify-email';
