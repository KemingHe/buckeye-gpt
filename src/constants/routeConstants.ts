// Stack Auth routes.
export const AUTH_SIGN_IN_ROUTE = "/auth/sign-in";
export const AUTH_SIGN_UP_ROUTE = "/auth/sign-up";
