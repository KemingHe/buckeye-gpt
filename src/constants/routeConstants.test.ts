import { describe, expect, it } from "vitest";

import * as routeConstants from "./routeConstants";

describe("Route constants", (): void => {
  it("matches the snapshot", (): void => {
    expect(routeConstants).toMatchSnapshot();
  });
});
