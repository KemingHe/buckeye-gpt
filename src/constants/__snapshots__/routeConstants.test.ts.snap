// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Route constants > matches the snapshot 1`] = `
{
  "AUTH_SIGN_IN_ROUTE": "/auth/sign-in",
  "AUTH_SIGN_UP_ROUTE": "/auth/sign-up",
}
`;
